torchao/quantization/quant_api.py

```diff
@@ -489,19 +492,80 @@ def int8_dynamic_activation_int8_semi_sparse_weight():
     """
     return int8_dynamic_activation_int8_weight(layout_type=SemiSparseLayoutType())
 
+
 def float8_weight_only(target_dtype: torch.dtype = torch.float8_e4m3fn):
     """
     Applies float8 weight-only symmetric per-channel quantization to linear layers.
+    
+    Args:
+        target_dtype (torch.dtype): The target data type for weight quantization. Default is torch.float8_e4m3fn.
```

Comment:
nit: might make sense to use `weight_dtype` as well to align with `float8_dynamic_activation_float8_weight`

Reply:
yeah will do